From 3ebc362fe3f70616922d46f3959c1e6d63cf76dc Mon Sep 17 00:00:00 2001
From: Dongdong Liu <liudongdong3@huawei.com>
Date: Tue, 11 Oct 2022 19:18:43 +0800
Subject: doc: fix application name in procinfo guide

[ upstream commit 2a65f12f6ab461e2cb81192c6bdd136191c82ea9 ]

In commit 996ef1176111 ("app: add all remaining apps to meson build"),
building the procinfo application through Meson has been done with a
binary name different from the previous Makefile build system [1].

When we dropped Makefile support, the documentation was not updated.

Fixes: 3cc6ecfdfe85 ("build: remove makefiles")
Cc: stable@dpdk.org

Signed-off-by: Dongdong Liu <liudongdong3@huawei.com>
Signed-off-by: David Marchand <david.marchand@redhat.com>
---
 doc/guides/tools/proc_info.rst | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/doc/guides/tools/proc_info.rst b/doc/guides/tools/proc_info.rst
index 9772d97ef0..e4f0c83f1b 100644
--- a/doc/guides/tools/proc_info.rst
+++ b/doc/guides/tools/proc_info.rst
@@ -1,10 +1,10 @@
 ..  SPDX-License-Identifier: BSD-3-Clause
     Copyright(c) 2015 Intel Corporation.
 
-dpdk-procinfo Application
-=========================
+dpdk-proc-info Application
+==========================
 
-The dpdk-procinfo application is a Data Plane Development Kit (DPDK) application
+The dpdk-proc-info application is a Data Plane Development Kit (DPDK) application
 that runs as a DPDK secondary process and is capable of retrieving port
 statistics, resetting port statistics, printing DPDK memory information and
 displaying debug information for port.
@@ -17,7 +17,7 @@ The application has a number of command line options:
 
 .. code-block:: console
 
-   ./<build_dir>/app/dpdk-procinfo -- -m | [-p PORTMASK] [--stats | --xstats |
+   ./<build_dir>/app/dpdk-proc-info -- -m | [-p PORTMASK] [--stats | --xstats |
    --stats-reset | --xstats-reset] [ --show-port | --show-tm | --show-crypto |
    --show-ring[=name] | --show-mempool[=name] | --iter-mempool=name ]
 
@@ -72,14 +72,14 @@ by name. For invalid or no mempool name no elements are displayed.
 Limitations
 -----------
 
-* dpdk-procinfo should run alongside primary process with same DPDK version.
+* dpdk-proc-info should run alongside primary process with same DPDK version.
 
-* When running ``dpdk-procinfo`` with shared library mode, it is required to
+* When running ``dpdk-proc-info`` with shared library mode, it is required to
   pass the same NIC PMD libraries as used for the primary application. Any
   mismatch in PMD library arguments can lead to undefined behavior and results
   affecting primary application too.
 
-* Stats retrieval using ``dpdk-procinfo`` is not supported for virtual devices like PCAP and TAP.
+* Stats retrieval using ``dpdk-proc-info`` is not supported for virtual devices like PCAP and TAP.
 
-* Since default DPDK EAL arguments for ``dpdk-procinfo`` are ``-c1, -n4 & --proc-type=secondary``,
+* Since default DPDK EAL arguments for ``dpdk-proc-info`` are ``-c1, -n4 & --proc-type=secondary``,
   It is not expected that the user passes any EAL arguments.
-- 
2.23.0

